array-manipulation-2/src/chunk.js

```diff
@@ -0,0 +1,41 @@
+// /* exported chunk */
+
+// count the length of number (size) return that number into new subarray, and if that size can be counted more than once (aka [1], [2]) then return the amount, and finish with pushing any remainders into the new subarray
```

Comment:
Is this your pseudocode? Why is there only one line? You should have a line for each step you take.